### PR TITLE
[Refactor] Naming update

### PR DESCRIPTION
**Pull request by DarkShadeKnigh**
# Pull request motivation(s)

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### CODE REVIEW

**Comment by DarkShadeKnigh** [comment]
Had a name change this caused issues with the search bar not being updated

File: configs/bllFlow.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-  "index_name": "bllFlow",
+  "index_name": "bllflow",
```
Comment (s-pace) [inline configs/bllFlow.json]: You can’t change it this way. We will have to provide you new credentials

Comment (DougManuel) [reply configs/bllFlow.json]: We'd appreciate the new credentials. Do we make a request by email? Or do you send them to the original email (they were originally sent to me).

Comment (s-pace) [reply configs/bllFlow.json]: Please apply and reply with a link to this thread. I will handle it asap when I am back from PTO in 2 weeks

**Comment by s-pace** [comment]
@DarkShadeKnigh solved. New indexName: bllflow, new ApiKey:ebce673983a3972ea16324f81b978cdc.

**Pull request closed without merging**